assemblies/assembly-audit-log.adoc

```diff
@@ -35,3 +35,5 @@ Audit logs are not forwarded to the internal log store by default because this d
 * For more information, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[{ocp-brand-name}  - About Logging]
 
 include::modules/getting-started/con-audit-log-config.adoc[leveloffset=+1]
```

Comment:
Similarly, 'con-audit-log-config.adoc' is a conceptual module, but its heading starts with a gerund, which typically indicates a procedural module. Should we change that as well, or is a separate Jira needed?

Reply:
You are right about procs generally beginning with gerunds and cons generally beginning with nouns, but I don't think this is a necessary change. It is also highly likely that this assembly will provide some procedural steps in the future, we just aren't there yet.